src/kubernetes/server/server-config-rabbit.yaml

```diff
@@ -1,44 +0,0 @@
-apiVersion: v1
```

Comment:
Since now this is gone and the `server-config-kafka.yaml` is renamed to `server-config.yaml`, so I guess there's nothing middleware related configs in the config file? Perhaps only Task/datasource related?

Reply:
correct, when we removed the `environmentVariables` param in: https://github.com/spring-cloud/spring-cloud-dataflow/commit/5f896f5f074c354001d29e971ab6e69af124a851

the configs became identical (or my 👀 are off ;-)) - so made sense just to combine them for now